Documentation/foo.txt

```diff
@@ -0,0 +1,20 @@
+ID uint16
```

Comment:
What is `foo.txt`?

Reply:
I did `git add -u Documentation/` and my laziness got punished - fixed now. Sorry about that!